deployments/common/image/base.yml

```diff
@@ -1,8 +1,17 @@
 channels:
-  - manics
+  #- manics
   - conda-forge
   - http://ssb.stsci.edu/astroconda
+  - defaults
```

Comment:
How do you figure out order between conda-forge and defaults?

Reply:
Unfortunately, trial and error.